### DISCUSSION TITLE
Ubuntu -22.04.2-LTS: Getting incorrect mimetype for .js file as text/javascript

### DISCUSSION THREAD

**Discussion by swetayadav1** [actions/virtual-environments#7262] [General questions]
**Current runner version:** '2.302.1'
**Operating System:** Ubuntu -22.04.2-LTS
**Runner Image:** Image: ubuntu-22.04 , Version: 20230227.2

Getting incorrect mimetype for .js file as text/javascript
It should be - **application/javascript**

We were getting correct mimetype with Ubuntu-20.04.5

Our pipeline is failing due to this issue. Please suggest

**Comment by magnetikonline** [comment]
See here: https://www.rfc-editor.org/rfc/rfc9239.html

> This document defines equivalent processing requirements for the various script media types. The most widely supported media type in use is text/javascript; all others are considered historical and obsolete aliases of text/javascript.

> The media type registrations herein are divided into two major categories: (1) the sole media type "text/javascript", which is now in common usage and (2) all of the media types that are obsolete (i.e., "application/ecmascript", "application/javascript", "application/x-ecmascript", "application/x-javascript", "text/ecmascript", "text/javascript1.0", "text/javascript1.1", "text/javascript1.2", "text/javascript1.3", "text/javascript1.4", "text/javascript1.5", "text/jscript", "text/livescript", and "text/x-ecmascript").

TL;DR - `application/javascript` is now obsolete - `text/javascript` is now the agreed MIME type.
